packages/xsnap/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@agoric/xsnap",
-  "version": "0.14.2",
-  "description": "Description forthcoming.",
+  "version": "0.14.3-u12.0",
```

Comment:
@michaelfig and @mhofman should we bump the version here or leave it at `0.14.2`?

Reply:
Oh right, please match any versions to what is currently on master

Reply:
Please revert this change